app/lib/communicator/communicator.js

```diff
@@ -223,6 +225,28 @@ function getSubmissionUrl( server ) {
     return ( server.lastIndexOf( '/' ) === server.length - 1 ) ? `${server}submission` : `${server}/submission`;
 }
 
+/**
+ * @param {Record<string, string | string[]>} [headers]
+ * @param {import('express').Request} [currentRequest]
+ */
+const getUpdatedRequestHeaders = ( headers = {}, currentRequest = getCurrentRequest() ) => {
+    const clientUserAgent = currentRequest?.headers['user-agent'];
+    const serverUserAgent = `Enketo/${config.version}`;
```

Comment:
Could `Enketo` be pulled from somewhere rather than being static here?

Reply:
Capturing discussion from Slack: We can pull this from `package.json` if you’d prefer, or introduce a more specific config if there’s a compelling reason. But the former also prompted me to realize that `version` does not account for git refs on forks, which would be an issue when deploying unmerged changes for testing.

Reply:
Let's keep as-is for now.